lib/mutant/bootstrap/rails.rb

```diff
@@ -0,0 +1,49 @@
+module Mutant
+  module Bootstrap
+    class Rails
+      include Procto.call, Concord.new(:world, :config)
+
+      def call
+        world.stdout.puts('Loading mutant config from rails environment')
+
+        # TODO add env to world.
+        #
+        # Make actual env configurable
+        ENV['RAILS_ENV'] = 'test'
+
+        world.kernel.require('./config/environment.rb')
+
+        ::Rails.application.eager_load!
+
+        # TODO: preload engines, these *love* to pack things
+        # in the main project
+        #
+        # TODO: allow custom preload hooks.
+
+        add_rails_matchers
+      end
+
+    private
+
+      def add_rails_matchers
+        # This logic sucks, instead rails should be come a match expression
+        # possibly allow match expressions like: `ActionController.subclasses`
```

Comment:
So expressions of the form `$Constant.$method` would expect to call a method on an object that returns the actual subclasses.

And: `$Constant{,.$method}` would match both the constant and the classes returned by the subclass.

We'd than still need an expression for the "entirety" of rails owned constants which could be: `<Rails>` or similar.